### PR TITLE
 Add k8s monitoring watchers

### PR DESCRIPTION
**Pull request by ChrsMark**
This PR adds the k8s watchers created for https://github.com/elastic/integrations/issues/4997 along with an installer script. 
These can be used for either testing or demos.

### CODE REVIEW

File: scripts/alerting/install_watchers.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                       type=str,
+                       nargs="?",
+                       const="changeme",
+                       default="changeme",
+                       help='The password to access the ES API')
+
+args = my_parser.parse_args()
+
+watchers_path = "./watchers"
+watchers = [f for f in listdir(watchers_path) if isfile(join(watchers_path, f))]
```
Comment (gizas) [inline scripts/alerting/install_watchers.py:43]: Can we add the `.disabled` extension in files? So if a use wants he can choose to install only specific ones?

File: scripts/alerting/install_watchers.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    watcher_name = watcher.split(".json")[0]
+    url = base_url+watcher_name+"?pretty"
+    res = requests.post(
+        url+"",
+        headers=headersList,
+        verify=False,
+        json=watcher_body,
+        auth = HTTPBasicAuth(args.username, args.password)
+    )
+    if res.status_code not in [200, 201]:
```
Comment (gizas) [inline scripts/alerting/install_watchers.py:64]: Why 201?

**Review by gizas — APPROVED** [review]

File: scripts/alerting/install_watchers.py (hunk 2 above)
Comment (ChrsMark) [reply scripts/alerting/install_watchers.py:64]: ES returns `201` when the Object is created for first time: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201
And 200 when it is updated (you `Put` the watch for 2nd, 3rd time etc).

So we need to accept both of them.

File: scripts/alerting/install_watchers.py (hunk 1 above)
Comment (ChrsMark) [reply scripts/alerting/install_watchers.py:43]: Nice idea, I will include it.

**Pull request merged**